lib/rspec/rails/fixture_support.rb

```diff
@@ -50,6 +50,12 @@ def self.proxy_method_warning_if_called_in_before_context_scope(method_name)
             end
           end
 
+          if ::ActiveRecord.version >= Gem::Version.new('6.1.0')
```

Comment:
Sorry you can't use `Gem::Version`, as we don't have a dependency on rubygems being available. A simple
```suggestion
          if Rails.version.to_f > 6.0
```
is preferred.

Reply:
Rubygem is part of the standard library since ruby 1.9. I can use `Rails.version.to_f` if you really want to but I don't see any real reason

Reply:
We don't load any part of the standard library unless 100% necessary as it causes environment poisoning (where someone might not have required the right file but we do, tests pass, but it fails outside of tests). There is also a speed benefit to not using rubygems which we ourselves utilise in our tests by disabling it (which gives us the speed benefit and tests we're not poisoning the env).

Reply:
e.g Its simpler to not use it so please don't :)

Reply:
Ok I'll use `Rails.version.to_f`, thanks for quick answer :)

Not trying to argue on such a small detail but `Rails.version` already loads rubygem 🤷‍♂ . But anyway having the syntax you proposed will make things consistent since that's what it's used in this library.